wcc-contentful-app/lib/generators/wcc/templates/menu/generated_add_menus.ts

```diff
@@ -174,19 +264,124 @@ export = function (migration: Migration) {
     validations: [],
     disabled: false,
     omitted: false,
-    items:
+    items: {
+      type: 'Link',
+      validations: [
+        {
+          linkContentType: ['dynamicButton', 'menuButton']
```

Comment:
dynamicButton should not be included in the validation unless we're going to define it in the gem